scilpy/segment/voting_scheme.py

```diff
@@ -56,7 +96,7 @@ def __init__(self, config, atlas_directory, transformation,
 
     def _init_bundles_tag(self):
         """
-        Using all bundles in the configuration file and the input models
+        Using all bundles in the configuration file and the input models.
```

Comment:
If I understand the comment correctly, I don't think there should be a period there